src/components/BaseHead.astro

```diff
@@ -58,18 +60,22 @@ const resolvedImage = {
 <!-- Font preloads -->
 <link
 	rel="preload"
-	href="https://fonts-cdn.astro.build/Obviously/Obviously Normal/Web/Obviously-Regular.woff2"
+	href="/fonts/Obviously/Obviously Normal/Web/Obviously-Regular.woff2"
 	as="font"
 	type="font/woff2"
 	crossorigin
 />
 
 <link
 	rel="preload"
-	href="https://fonts-cdn.astro.build/MD IO/Web/MDIO0.5-Regular.woff2"
+	href="/fonts/MD IO/Web/MDIO0.5-Regular.woff2"
 	as="font"
 	type="font/woff2"
 	crossorigin
 />
 
 <link rel="preload" href={Inter} as="font" type="font/woff2" crossorigin />
+
+<!-- image preloads -->
+
+<link rel="preload" as="image" href="/assets/noise.png" />
```

Comment:
unrelated to fonts, noticed the noise image file could use a preload for a slight perf improvement